diff --git a/hybridauth/Hybrid/Providers/LinkedIn.php b/hybridauth/Hybrid/Providers/LinkedIn.php
index 137b2ab..6d3438e 100644
--- a/hybridauth/Hybrid/Providers/LinkedIn.php
+++ b/hybridauth/Hybrid/Providers/LinkedIn.php
@@ -40,6 +40,7 @@ class Hybrid_Providers_LinkedIn extends Hybrid_Provider_Model {
 				'date-of-birth',
 				'phone-numbers',
 				'summary',
+				'positions'
 			);
 		}
 
@@ -132,6 +133,11 @@ class Hybrid_Providers_LinkedIn extends Hybrid_Provider_Model {
 			$this->user->profile->email = (string) $data->{'email-address'};
 			$this->user->profile->emailVerified = (string) $data->{'email-address'};
 
+			if ($data->{'positions'}) {
+        $this->user->profile->job_title = (string) $data->{'positions'}->{'position'}->{'title'};
+        $this->user->profile->organization_name = (string) $data->{'positions'}->{'position'}->{'company'}->{'name'};
+      }
+
 			if (isset($data->{'picture-url'})) {
 				$this->user->profile->photoURL = (string) $data->{'picture-url'};
 
diff --git a/hybridauth/Hybrid/User_Profile.php b/hybridauth/Hybrid/User_Profile.php
index 6a4070c..403be89 100644
--- a/hybridauth/Hybrid/User_Profile.php
+++ b/hybridauth/Hybrid/User_Profile.php
@@ -149,4 +149,15 @@ class Hybrid_User_Profile {
 	 */
 	public $zip = null;
 
+  /**
+   * Job title
+   * @var string
+   */
+  public $job_title = null;
+
+  /**
+   * Organization name
+   * @var string
+   */
+  public $organization_name = null;
 }
